docs/en/inference/pipeline.md

```diff
@@ -116,6 +116,25 @@ for item in pipe.stream_infer(prompts, gen_config=gen_config):
     print(item)
 ```
 
+- **An example to cauculate logits & ppl:**
+
+```python
+from transformers import AutoTokenizer
+from lmdeploy import pipeline
+pipe = pipeline('internlm/internlm2-chat-7b')
```

Comment:
```
model_repoid_or_path='internlm/interlm2-chat-7b'
pipe = pipeline(model_repoid_or_path)
tokenizer = AutoTokenizer.from_pretrained(model_repoid_or_path, trust_remote_code=True)
```
In this way, user can only change the `model_repoid_or_path` to test their interested models